server/tests/participant.form.test.js

```diff
@@ -0,0 +1,242 @@
+const { readFileSync } = require('fs');
+const { join } = require('path');
+const { ValidationError } = require('yup');
+const { startDB, closeDB } = require('./util/db');
+const { parseAndSaveParticipants, getParticipants } = require('../services/participants.js');
+const { evaluateBooleanAnswer } = require('../validation');
+
+describe('Participants Service', () => {
+  beforeAll(async () => {
+    await startDB();
+  });
+
+  afterAll(async () => {
+    await closeDB();
+  });
+
+  const allParticipants = [
```

Comment:
Might make sense to keep this in a JSON file in the same location as our test .xlsx fikes.

Reply:
I feel that we do too many tweaks over the test data currently to adapt to the tests result, we can add that json for upcoming PRs if the sheet won’t change too much